libraries/browser-tracker-core/src/tracker/out_queue.ts

```diff
@@ -351,7 +352,14 @@ export function OutQueueManager(
       // Time out POST requests after connectionTimeout
       const xhrTimeout = setTimeout(function () {
         xhr.abort();
-        executingQueue = false;
+
+        if (retryFailures) {
+          LOG.warn(`Request failed, retrying.`);
+        } else {
+          LOG.error(`Request failed and retryFailures is false, will not retry.`);
+          removeEventsFromQueue(numberToSend);
+          executingQueue = false;
```

Comment:
Might be missing something here, but should we set `executingQueue = false` also if `retryFailures` is true?